doc-content/jbpm-docs/src/main/asciidoc/BAM/Dashbuilder-data-management-con.adoc

```diff
@@ -1,23 +1,23 @@
 [id='Dashbuilder-data-management-con_{context}']
-= Dashbuilder data management in {CENTRAL}
+= Dashboard data management in {CENTRAL}
 
 ifdef::PAM,JBPM[]
 
-Dashbuilder is a dashboard and reporting tool integrated in {CENTRAL} and is used by the Datasets editor and Content Manager page. Dashbuilder supports the following data types:
+{CENTRAL} contains dashboard and reporting tools that you can access from the data sets editor and Content Manager page. You can use these tools to work with the following types of items:
```

Comment:
Is the term "Content Manager page" still used and known to users?

Reply:
Yes, I would prefer to use "Page Editor" here. When there's no page in the BC, there's the "Content Management" title, but creating pages is done in the "Page Editor"